mixin/alerts/receive.libsonnet

```diff
@@ -56,10 +55,14 @@
                 sum by (job) (rate(thanos_receive_forward_requests_total{result="error", %(selector)s}[5m]))
               /
                 sum by (job) (rate(thanos_receive_forward_requests_total{%(selector)s}[5m]))
-              * 100 > %(forwardErrorThreshold)s
+              )
+              >
+              (
+                max by (job) ((thanos_receive_replication_factor+1) / 2)
```

Comment:
If I'm understanding this right, then I think you need to use `floor` on this, as we are not doing integer divisions here.